logic/randomize.py

```diff
@@ -199,9 +199,10 @@ def randomize_starting_items(self):
                 starting_items.add(item)
 
         if self.options["random-starting-item"]:
-            possible_random_starting_items = tuple(
-                set(RANDOM_STARTING_ITEMS) - set(self.options["starting-items"])
-            )
+
+            possible_random_starting_items = RANDOM_STARTING_ITEMS
+            for starting_item in self.options["starting-items"]:
+                possible_random_starting_items.remove(starting_item)
```

Comment:
```suggestion
            possible_random_starting_items = [
                item
                for item in RANDOM_STARTING_ITEMS
                if item not in self.options["starting-items"]
            ]
```

Otherwise it produces an error when you start with multiple beetles for example, also don't modify the global variable, this will break once you roll multiple seeds